all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -57,7 +57,7 @@ objects:
       spec:
           containers:
             - name: ${JAEGER_NAME}
-              image: jaegertracing/all-in-one:f209812d
+              image: jaegertracing/all-in-one:9f3984cd
```

Comment:
When a new version of `uber/jaeger` is released it will automatically build new Docker images. These templates do not directly depend on the exact versions of Jaeger backend, so maybe they should take the "version" as a parameter (and default to latest version)? Don't know if it's possible.